### PR TITLE
release: fix Elementary support

### PR DESCRIPTION
**Pull request by zyga**
I just realized that Elementary Loki 0.4 doesn't really work with
snappy. The root cause seems to be that the /etc/os-release ID field is
"elementary OS", not "elementary" as can be see here:

https://github.com/zyga/os-release-zoo/blob/master/elementary/elementary-0.4.txt

This patch fixes the issue by adding the missing " OS" part where it
matters.

Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

**Comment by chipaca** [comment]
I like the colour of this bikeshed.

File: release/release.go [hunk 1]
```diff
@@ -45,8 +45,7 @@ func (os *OS) ForceDevMode() bool {
 		fallthrough
 	case "ubuntu":
 		return false
-
-	case "elementary":
```
Comment (niemeyer) [inline release/release.go]: Please fix it in the opposite direction: strings.TrimPrefix(s, " OS").

Comment (niemeyer) [inline release/release.go]: Actually, even better: strings.Fields(s)[0].  Let's pick just the first token there.

File: release/release_test.go [hunk 2]
```diff
@@ -109,8 +109,8 @@ func (s *ReleaseTestSuite) TestForceDevMode(c *C) {
 		// Please keep this list sorted
 		{id: "arch", devmode: true},
 		{id: "debian", devmode: true},
-		{id: "elementary", devmode: true},
-		{id: "elementary", idVersion: "0.4", devmode: false},
+		{id: "elementary OS", devmode: true},
+		{id: "elementary OS", idVersion: "0.4", devmode: false},
```
Comment (niemeyer) [inline release/release_test.go]: This should never have spaces: _A lower-case string (no spaces or other characters outside of 0–9, a–z, ".", "_" and "-")_

Comment (zyga) [inline release/release_test.go]: I've reported the bug upstream. It was assigned and should be fixed soon.

File: release/release.go (hunk 1 above)
Comment (zyga) [inline release/release.go]: I've updated this to just handle both "elementary" and "elementary OS"

**Comment by chipaca** [comment]
`I APPROVE OF THIS`

File: release/release_test.go [hunk 3]
```diff
@@ -111,6 +111,8 @@ func (s *ReleaseTestSuite) TestForceDevMode(c *C) {
 		{id: "debian", devmode: true},
 		{id: "elementary", devmode: true},
 		{id: "elementary", idVersion: "0.4", devmode: false},
+		{id: "elementary OS", devmode: true},
+		{id: "elementary OS", idVersion: "0.4", devmode: false},
```
Comment (niemeyer) [inline release/release_test.go]: Again, this is bogus. Let's not accept broken IDs.

Please use this logic for parsing the id out:
```
id = strings.Fields(strings.ToLower(id))[0]
```

**Pull request closed without merging**